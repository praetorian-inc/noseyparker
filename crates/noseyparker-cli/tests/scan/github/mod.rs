use super::*;

#[test]
fn github_all_orgs_no_api_url() {
    let scan_env = ScanEnv::new();
    assert_cmd_snapshot!(noseyparker_failure!(
        "scan",
        "-d",
        scan_env.dspath(),
        "--all-github-organizations"
    ));
}

#[test]
fn github_all_orgs_explicit_default_api_url() {
    let scan_env = ScanEnv::new();
    assert_cmd_snapshot!(noseyparker_failure!(
        "scan",
        "-d",
        scan_env.dspath(),
        "--all-github-organizations",
        "--github-api-url",
        "https://api.github.com"
    ));
}
