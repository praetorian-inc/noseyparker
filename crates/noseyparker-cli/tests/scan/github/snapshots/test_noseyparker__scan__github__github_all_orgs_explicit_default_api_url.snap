---
source: crates/noseyparker-cli/tests/scan/github/mod.rs
expression: status
---
exit status: 2
